packages/app-storage/src/Query.tsx

```diff
@@ -41,11 +43,51 @@ class Query extends React.PureComponent<Props, State> {
         value as Storage$Key$Value
       );
 
-      cache[id] = withStorageDiv(key, { params: values })(
-        (value: any) =>
-          valueToText(key.type, value),
-        { className: 'ui--output' }
-      );
+      // FIXME - Move into ui-app/src/Params/valueToAccountList for reuse
```

Comment:
The idea is right - however, we should actually be extending https://github.com/polkadot-js/apps/blob/master/packages/ui-app/src/Params/valueToText.ts to return React.Node (or text, text just being a subset of Node)